src/index.js

```diff
@@ -38,72 +28,52 @@ class HaStore extends EventEmitter {
     this.queue = queue(
       this.config,
       this,
-      this.store,
+      this.store
     );
   }
 
-  /**
-   * Gets a list of records from source
-   * @param {string|number|array<string|number>} ids The id of the record to fetch
-   * @param {object} params (Optional)The Request parameters
-   * @returns {Promise} The eventual single record
-   */
-  async get(ids, params = {}, agg = null) {
+  get(ids, params = {}, agg = null) {
+    if (params === null) params = {};
+    const key = contextKey(this.config.delimiter, params);
+    return this.queue.getHandles(key, [ids], params, agg)
+      .then(handles => handles[0]);
+  }
```

Comment:
[⏳] Singular word `id` helps undestanding the context.

Maybe, if it's an array, we could shift the array and grab the first value for extra safety.

```suggestion
  get(id, params = {}, agg = null) {
    if (params === null) params = {};
    const key = contextKey(this.config.delimiter, params);
    return this.queue.getHandles(key, [id], params, agg)
      .then(handles => handles[0]);
  }
```